### PR TITLE
Create clinical-gastroenterology-and-hepatology.csl

### PR DESCRIPTION
**Pull request by zuphilip**
See https://forums.zotero.org/discussion/56355/style-request-clinical-gastroenterology-and-hepatology/#Comment_245501

### CODE REVIEW

**Comment by zuphilip** [comment]
I don't understand:
```
  1) independent style clinical-gastroenterology-and-hepatology is a valid CSL 1.0 style

     Failure/Error: expect(CSL.validate(path)).to eq([])

       

       expected: []

            got: [[13, "Expecting an element name, got nothing"], [13, "Invalid sequence in interleave"], [13, "Element contributor failed to validate content"], [0, "Extra element contributor in interleave"], [13, "Element info failed to validate content"]]
```

File: clinical-gastroenterology-and-hepatology.csl [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    <title>Clinical Gastroenterology and Hepatology</title>
+    <id>http://www.zotero.org/styles/clinical-gastroenterology-and-hepatology</id>
+    <link rel="self" href="http://www.zotero.org/styles/clinical-gastroenterology-and-hepatology"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
+    <link href="http://www.cghjournal.org/content/authorinfo" rel="documentation"/>
+    <author>
+      <name>Philipp Zumstein</name>
+      <uri>https://github.com/zuphilip/</uri>
+    </author>
+    <contributor/>
```
Comment (adam3smith) [inline clinical-gastroenterology-and-hepatology.csl]: the validator doesn't like the empty contributor node

**Comment by adam3smith** [comment]
thanks!

**Comment by zuphilip** [comment]
Thank you for translating Sheldon-speak to something I can understand :-)

**Comment by adam3smith** [comment]
For future reference:
> 1) independent style clinical-gastroenterology-and-hepatology is a valid CSL 1.0 style
is always validation. and then the actual error (got is a list of value pairs, where the first one is a line number and the second one an error description. That's not always helpful, but in this case the line number was enough to look in the right place.

**Pull request merged**